wizard_builder/models.py

```diff
@@ -29,8 +23,24 @@ class PageBase(PolymorphicModel):
         (WHO, 'Who'),
     )
 
-    section = models.IntegerField(choices=SECTION_CHOICES,
-                                  default=WHEN)
+    position = models.PositiveSmallIntegerField("position", default=0)
+    section = models.IntegerField(choices=SECTION_CHOICES, default=WHEN)
+    site = models.ForeignKey(Site, null=True)
+    objects = PageBaseManager()
+
+    def add_site_from_site_id(self):
+        if getattr(settings, 'SITE_ID'):
+            self.site_id = settings.SITE_ID
+
+    def set_page_position(self):
+        cls = self.__class__
+        if bool(cls.objects.exclude(pk=self.pk)):
+            self.position = cls.objects.exclude(pk=self.pk).latest('position').position + 1
+
+    def save(self, *args, **kwargs):
+        self.add_site_from_site_id()
+        self.set_page_position()
```

Comment:
ah!!! yea it will

I added some `not self.attr`s so that won't override defaults. Will add some tests